sandbox/apparmor/notify/listener/listener_test.go

```diff
@@ -862,6 +868,7 @@ func (*listenerSuite) TestRunConcurrency(c *C) {
 
 	// Closes the sendChan
 	restoreEpollIoctl()
+	epollIoctlRestored = true
```

Comment:
you can more safely set it before the call as you are about to do exactly the thing controlled by the flag in the defer

Reply:
Thanks! I'll fix this in the rebase to clean up the history a bit.